client/stylesheets/abstracts/_mixins.scss

```diff
@@ -47,3 +47,27 @@
 	outline: 2px solid transparent;
 	outline-offset: -2px;
 }
+
+// Sets grid positions for children of grid elements.
+@mixin set-item-grid-position( $breakpoint, $items_class, $number_of_items ) {
```

Comment:
What do you think of changing `$breakpoint` to `$wrap-after`, or `per-line`?  I keep thinking this means responsive breakpoint, rather than the count-per-line of items.

Reply:
I like it, much better!